.github/workflows/extended.yml

```diff
@@ -67,25 +67,45 @@ jobs:
           fetch-depth: 1
       - name: Free Disk Space (Ubuntu)
         uses: jlumbroso/free-disk-space@54081f138730dfa15788a46383842cd2f914a1be
-      - name: Install Rust
+      - name: Setup Rust toolchain
         run: |
-          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
-          source $HOME/.cargo/env
+          rustup toolchain install stable
```

Comment:
I don't think it needs to install stable toolchain (maybe this is a merge mistake 🤔 )